### PR TITLE
Update Hydra image to latest stable release

### PR DESCRIPTION
**Pull request by yashutanna**
Resolves: https://github.com/reactioncommerce/reaction-feature-requests/issues/112#issue-484468960
Impact: **minor**
Type: **chore**

## Issue
use the stable version of hydra as opposed to beta

## Solution
this PR uses ENV variables as expected by the stable version of hydra as old values were deprecated

## Testing
1. update reaction with this [PR](https://github.com/reactioncommerce/reaction/pull/5640#issue-323714320)
2) authenticate as normal

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ networks:
 services:
 
   hydra-migrate:
-    image: oryd/hydra:v1.0.0-beta.9-alpine
-    command: migrate sql -e
+    image: oryd/hydra:latest
```
Comment (ticean) [inline docker-compose.yml]: Please pin to the latest version.

Comment (ticean) [inline docker-compose.yml]: Please pin to the latest version.

File: docker-compose.yml [hunk 2]
```diff
@@ -8,15 +8,15 @@ networks:
...
+    command: migrate sql -e -y
     depends_on:
       - postgres
     env_file: ".env"
     restart: on-failure
 
   hydra:
-    image: oryd/hydra:v1.0.0-beta.9-alpine
+    image: oryd/hydra:latest
```
Comment (ticean) [inline docker-compose.yml]: Please pin to the latest version.

**Comment by aldeed** [comment]
I have manually merged these changes into the `release-v3.0.0` branch and tested

**Pull request closed without merging**